run.sh

```diff
@@ -159,3 +162,4 @@ yum remove -y autoconf automake gcc gcc-c++ git libtool nasm  zlib-devel tar xz
 yum clean all
 rm -rf /var/lib/yum/yumdb/*
 echo "/usr/local/lib" > /etc/ld.so.conf.d/libc.conf
+mkdir -p /mnt/tmp
```

Comment:
What is it for ?

The Dockerfile uses :

 WORKDIR /tmp/workdir

Assuming all the files to be managed would be stored there